.github/workflows/elixir.yml

```diff
@@ -32,20 +31,65 @@ jobs:
       uses: actions/cache@v2
       with:
         path: deps
-        key: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-${{ hashFiles('**/mix.lock') }}
-        restore-keys: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-
+        key: test-${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-${{ hashFiles('**/mix.lock') }}
+        restore-keys: test-${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-
```

Comment:
strictly speaking, wouldn't it make more sense for the deps cache to not have this prefix (since it's used for the linter job as well), and do that on the build cache instead?

Reply:
Honestly, I just mindlessly wrote it everywhere, I think you're right